rust-guest/src/compile/scale.rs

```diff
@@ -0,0 +1,11 @@
+#![allow(unused_mut)]
+
+use crate::context::RunContext;
+use std::string::String;
+use super::generated::{Context, Request, Response};
```

Comment:
Why are all of these necessary? Doesn't the function just need the Context?

Reply:
Nope, forgot to remove. Yep, 100%.